exonum/CHANGELOG.md

```diff
@@ -88,6 +88,11 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - `ExonumJsonDeserialize` trait is implemented for `F32` and `F64`. (#461)
 
+### Internal improvements
+
+- Consensus messages are stored persistently (in the database), so restart will
+  not affect the node's behaviour. (#322)
```

Comment:
Nitpick: "behaviour" is the British spelling, the US one is "behavior".

Reply:
Yes, we've decided to use American english only